wordpress-mobile/WordPress-Android
# Fix 6951 - Aztec newlines lost with media success/failure processing

Fixes #6951

Related Aztec PR https://github.com/wordpress-mobile/AztecEditor-Android/pull/569
**NOTE:** the AztecEditor PR need to be merged first, and then we need to update the hash reference in `build.gradle` here.

To test:
1. start a new draft
2. write some text entering “ENTER” on purpose, like:
“something<ENTER>”
“another thing<enter>”
“bla bla<enter>”
“bla bla<enter>”
“bla bla<enter>”
“bla bla<enter>”

3. insert some media items
4. exit the editor while media items are uploading
5. either wait until uploads finish, or make the uploads fail, maybe by turning airplane mode ON (_please test both cases as it takes 2 different code paths_)
6. see the post show the error in the post list
8. now open the Post
9. see the Post is kept as originally saved

cc @0nko
